roles/container_runtime/tasks/systemcontainer_crio.yml

```diff
@@ -53,6 +53,8 @@
     name: "cri-o"
     image: "{{ l_crio_image }}"
     state: latest
+    values:
+    - "ADDTL_MOUNTS={{',' + (container_runtime_crio_additional_mounts | join(,)) if container_runtime_crio_additional_mounts != [] else '' }}"
```

Comment:
I believe you should use single quotes here. The internals of the ``container_runtime_crio_additional_mounts`` will need to be using double quotes.

Reply:
Sorry, ran a few iterations before it arrived at this solution.  I like the incoming array to be strings that get joined by a ','.  This is much easier to process in an inventory file.